webdriver/bidi/emulation.json

```
@@ -260,6 +256,38 @@
               }
             }
           },
+          "cross_origin_iframes": {
+            "__compat": {
+              "description": "Applies override to newly created cross-origin iframes.",
+              "support": {
+                "chrome": {
+                  "version_added": false
+                },
+                "chrome_android": "mirror",
+                "edge": "mirror",
+                "firefox": {
+                  "version_added": "143",
+                  "notes": "For new cross-origin iframes the override was not applied."
```

This note has two issues:

- It uses past tense, but it is not clear what version(s) it refers to.
- It seems to describe exactly what the subfeature description describes.

So I think we can just remove it.

```suggestion
                  "version_added": "143"
```